service.go

```diff
@@ -121,6 +121,33 @@ func (c *Client) ListServices(o ListServiceOptions) (*ListServiceResponse, error
 	return &result, c.decodeJSON(resp, &result)
 }
 
+// ListServices lists existing services processing paginated responses
+func (c *Client) ListServicesPaginated(ctx context.Context, o ListServiceOptions) ([]Service, error) {
+	services := make([]Service, 0)
```

Comment:
I totally missed this before because we don't have automatic linter checks turned on yet, but my editor flagged this. Mind converting this to `var services []Service`? The `append()` below will take care of allocation of the slice.

Reply:
I think they're equivalent. Is there a style guide you're following?

Reply:
Not a specific style guide, just a set of linters I've been running on my stuff for years. Never had an issue with it so I tend to fix things it yells about. 

Thank you for the fix. :bow: